src/fragments/lib/storage/flutter/list.mdx

```diff
@@ -53,16 +33,21 @@ Future<void> listAlbum() async {
 
 ```
```

Comment:
Can't add a note on the line but please align all examples when it comes to exception handling. In other storage examples, something like the following pattern was being used:

```dart
try {
  // ...
} on StorageException (e) {
  safePrint('Error performing <operation name>: ${e.message}');
  rethrow;
}
```